angular-holiday-planner/src/app/services/auth/auth.service.ts

```diff
@@ -17,15 +17,26 @@ export class AuthService {
   user: User | null | undefined;
   loggedIn = false;
   constructor(private auth: Auth) {
-      this.user = JSON.parse(sessionStorage.getItem("firebase:authUser:AIzaSyDqnu4jzVtNQ1ywyoFsZCskGZd5jvXAGH4:[DEFAULT]") ?? "")
-  }
+    console.log(sessionStorage.key(0))
+    const storage = sessionStorage.key(0);
+
+    if(storage){
+      this.user = JSON.parse(sessionStorage.getItem(storage) ?? "")
+
+    }
+ }
 
   registerUser(user: IUser){
     return this.auth.setPersistence(browserSessionPersistence).then(() => createUserWithEmailAndPassword(this.auth, user.email, user.password))
   }
 
   login(user: IUser){
-    return this.auth.setPersistence(browserSessionPersistence).then(() => signInWithEmailAndPassword(this.auth, user.email, user.password).then(res => this.user = res.user));
+    return this.auth.setPersistence(browserSessionPersistence).then(() => signInWithEmailAndPassword(this.auth, user.email, user.password).then(res => {
+      // console.log(this.auth.currentUser)
```

Comment:
Please remove commented out code before merge.

Reply:
They have been removed in a different pull request